docs/root/configuration/overview/v2_overview.rst

```diff
@@ -543,11 +564,11 @@ the shared ADS channel.
 Management Server Unreachability
 --------------------------------
 
-When Envoy instance looses connectivity with the management server, Envoy will latch on to
-the previous configuration while actively retrying in the background to reestablish the 
-connection with the management server. 
+When Envoy instance loses connectivity with the management server, Envoy will latch on to
```

Comment:
While we're here, it should read "When an Envoy instance loses..."

Reply:
Thanks, @zuercher! This is addressed in 75a1537.